lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -207,3 +235,26 @@ func loadAPNs(gateway *protos.Identity_Gateway) (map[string]*lte_models.ApnConfi
 
 	return apnsByName, apnResources, nil
 }
+
+// appendApnResourcesToSubProtos adds the gateway-specific apn resources data to subscriber protos before
+// returning to AGWs.
+func appendApnResourcesToSubProtos(subProtos []*lte_protos.SubscriberData, apnResources lte_models.ApnResources) []*lte_protos.SubscriberData {
```

Comment:
Rather than taking the apnResources as an arg, can the fn load the apnResources itself?